### PR TITLE
Feature/restaurar contraseña

### PR DESCRIPTION
**Pull request by marnunrey2**
For making sure that the feature works correctly, we need to run the following commands:
```
pip install -r requirements.txt
python decide/manage.py makemigrations
python decide/manage.py migrate
```

In your .env add the following (ask marnunrey2 for the values of the variables and do not share them):
```
EMAIL_HOST_USER = "email_host_user_marnunrey2"
EMAIL_HOST_PASSWORD = "email_host_password_marnunrey2"
```

If you did all of this correctly, it should work.

- Go to the login page and click on the restore password link
- This will send you to a page asking for your email (recall the email must be from a user that already exists)
- Then, a page telling you the instructions to follow will appear
- Go to your email and click on the link attached
- It will appear a page to change your password (recall you must introduce a correct password as in the registration form)
- Finally, a complete page will appear informing you that you can now access the login with the new password

### CODE REVIEW

**Review by FelixoGudiel — APPROVED** [review]
Good job, the functionality works just as intended and the 4 tests you added all run accordingly.

File: decide/authentication/test-selenium.py [hunk 1]
```diff
@@ -115,6 +147,11 @@ def test_nocaptcha(self):
         )
 
 
+"""
```
Comment (JaviGarc1a) [inline decide/authentication/test-selenium.py]: Can you delete?

**Comment by JaviGarc1a** [comment]
image
When I used an old link, the error text appeared in English, you should keep only one language.

**Review by JaviGarc1a — APPROVED** [review]
Great job!

**Pull request merged**